dpctl/tests/test_sycl_kernel_submit.py

```diff
@@ -37,7 +37,7 @@ def test_create_program_from_source (self):
             size_t index = get_global_id(0);                                   \
             c[index] = d*a[index] + b[index];                                  \
         }"
-        with dpctl.device_context(dpctl.device_type.gpu, 0):
+        with dpctl.device_context("opencl:gpu:0"):
```

Comment:
Is it possible to set only `"gpu"` ? Is it possible to select default backend?

Reply:
I think we need rules for composing this string for developers and users to follow.

Reply:
I did not add the option for default backend yet. Let us do as a follow up. Opened #100